articles/quickstart/native/wpf-winforms/index.yml

```diff
@@ -39,8 +39,9 @@ sdk:
   url: https://github.com/auth0/auth0-oidc-client-net
   logo: windows
 requirements:
-  - Microsoft Visual Studio 2017
-  - .NET Framework 4.6.2
+  - Microsoft Visual Studio 2022
+  - .NET Framework 4.8.1
```

Comment:
Bumped this to the lowest supported by microsoft. Our SDK still support lower versions tho, but let's keep it modern.